tests/cases/fourslash/codeFixSpellingShortName1.ts

```diff
@@ -0,0 +1,6 @@
+/// <reference path='fourslash.ts' />
+
+////export let abc = 1;
+////[|abC|] = 1;
+
+verify.rangeAfterCodeFix('abc');

```

Comment:
Does this exemplify the length exception? I thought the length had to be less than 3 characters.

Reply:
😂 yet another off-by-one error.  changed to `ab/aB`